facades/PC/src/main/java/org/terasology/engine/Terasology.java

```diff
@@ -77,7 +77,7 @@
  * <tr><td>-headless</td><td>Start headless.</td></tr>
  * <tr><td>-loadlastgame</td><td>Load the latest game on startup.</td></tr>
  * <tr><td>-noSaveGames</td><td>Disable writing of save games.</td></tr>
- * <tr><td>-noCrashReport</td><td>Disable crash reporting.</td></tr>
+ * <tr><td>-noCrashReport</td><td>Disable crash reporting when having a crash.</td></tr>
```

Comment:
No need to change the documentation - it still makes sense (the option disables crash reporting, but not issue/feedback reporting)